demangle/Cargo.toml

```diff
@@ -25,5 +26,11 @@ msvc-demangler = "0.8.0"
 rustc-demangle = "0.1.16"
 symbolic-common = { version = "7.3.3", path = "../common" }
 
+[dev-dependencies]
+symbolic = { path = "..", features = ["demangle"] }
```

Comment:
Just a heads up… I know this pattern is super useful, as I have also used it for the rust SDK, but you are not able to properly publish with this: https://github.com/rust-lang/cargo/issues/4242 😭

Reply:
Alright, let me revert this then. Too bad, it made the examples much nicer.

Reply:
Yes it definitely does! I’m also not yet sure what I should do about this for the SDK, since I depend on `sentry::init` and `sentry::test` a lot in the doctests 😭

Reply:
For the Sentry SDK, doing something similar to `futures` would probably be worth it. In symbolic, I guess people can extrapolate how to rewrite the imports until there's a proper solution for this in cargo.